dyn_em/module_first_rk_step_part1.F

```diff
@@ -714,6 +714,11 @@ SUBROUTINE first_rk_step_part1 (   grid , config_flags              &
      &        ,q2_ndg_new=grid%q2_ndg_new                  &
      &        ,sn_ndg_old=grid%sn_ndg_old                  &
      &        ,sn_ndg_new=grid%sn_ndg_new                  &
+     &        ,pxlsm_modis_veg=config_flags%pxlsm_modis_veg                   &
```

Comment:
@coastwx 
Was there ever an expectation of NMM using this scheme? We can go several ways: ifdefs and optional fields and present statements, or mods to the NMM Registry and changes to the NMM module_PHYSICS_CALLS.F